packages/elements/src/components/ino-accordion/ino-accordion.tsx

```diff
@@ -50,26 +50,34 @@ export class Accordion implements ComponentInterface {
     });
 
     const inoAccordionContent = classNames({
-      visibility: this.expanded ? 'visible' : 'hidden',
-      'ino-accordion--content': true,
+      'ino-accordion__content--visible': this.expanded,
+      'ino-accordion__content': true,
     });
 
     return (
       <div class={inoAccordionClasses}>
-        <div class="ino-accordion--header" onClick={() => this.toggleExpand()}>
-          <button class={this.expanded ? 'clicked' : ''}>
-            <span></span>
-            <span></span>
+        <div class="ino-accordion__header" onClick={() => this.toggleExpand()}>
+          <button
+            class={
+              this.expanded
+                ? 'ino-accordion__toggle-button--expanded'
+                : 'ino-accordion__toggle-button--collapsed'
+            }
+          >
+            <span class="toggle-icon"></span>
+            <span class="toggle-icon"></span>
           </button>
           <span
-            class={`ino-accordion--title ${this.expanded ? 'expanded' : ''}`}
+            class={`ino-accordion__title ${
+              this.expanded ? 'ino-accordion__title--expanded' : ''
+            }`}
```

Comment:
I would suggest to define the conditional class like this: https://github.com/inovex/elements/blob/e414aefaef9218dc11acc59f863d6ae6233d6bea/packages/elements/src/components/ino-chip/ino-chip.tsx#L140-L164